### PR TITLE
Intergration Crud Status Test Moved To App_spec

### PR DESCRIPTION
**Pull request by AshantiMutinta**
- [x] I have added a summary of my changes to the changelog
- [x] Moved handle_remove_ok into App_Spec
- [x] Moved handle_update_ok into App_Spec
- [x] Moved handle_remove_update_ok into App_Spec

### CODE REVIEW

File: app_spec/test/test.js [hunk 1]
```diff
@@ -116,39 +116,172 @@ scenario2.runTape('delete_entry_post', async (t, { alice, bob }) => {
...
+  t.ok(updatedPostv2Initial.Ok)
+  t.deepEqual(JSON.parse(updatedPostv2Initial.Ok.App[1]), updatePostContentV2)
+
+  //get v2 latest address
+  const updatedPostv2Latest = alice.call("blog", "get_post", { post_address: UpdateResultV2.Ok })
+  t.ok(updatedPostv2Latest.Ok)
+  t.deepEqual(JSON.parse(updatedPostv2Latest.Ok.App[1]), updatePostContentV2)
+
+
+   //get v2 using initial adderss
```
Comment (lucksus) [inline app_spec/test/test.js]: This getting V1, right?
```suggestion
   //get v1 using initial address
```

File: app_spec/test/test.js [hunk 2]
```diff
@@ -116,39 +116,172 @@ scenario2.runTape('delete_entry_post', async (t, { alice, bob }) => {
...
+   t.ok(GetInitialPostV2Latest.Ok)
+   t.deepEqual(JSON.parse(GetInitialPostV2Latest.Ok.App[1]),updatePostContentV2)
+
+  //update to version 3
+  const UpdatePostV3Content = { content: "Hello Holo V3", date_created: "now" };
+  const updateParamsV3 = { post_address: createResult.Ok, new_content: "Hello Holo V3" }
+  const UpdateResultV3 = await bob.callSync("blog", "update_post", updateParamsV3)
+  t.ok(UpdateResultV3.Ok)
+
+  //get v2 using initial adderss
```
Comment (lucksus) [inline app_spec/test/test.js]:
```suggestion
  //get v3 using initial adderss
```

File: app_spec/test/test.js [hunk 3]
```diff
@@ -116,39 +116,172 @@ scenario2.runTape('delete_entry_post', async (t, { alice, bob }) => {
...
+  const updatedPostV3Initial = alice.call("blog", "get_post", { post_address: createResult.Ok })
+  t.ok(updatedPostV3Initial.Ok)
+  t.deepEqual(JSON.parse(updatedPostV3Initial.Ok.App[1]), UpdatePostV3Content)
+
+  //get v2 latest address
```
Comment (lucksus) [inline app_spec/test/test.js]:
```suggestion
  //get v3 using address of v2
```

File: app_spec/test/test.js [hunk 4]
```diff
@@ -116,39 +116,172 @@ scenario2.runTape('delete_entry_post', async (t, { alice, bob }) => {
...
+
+     const entryWithOptionsGet = { post_address: createResult.Ok}
+     const entryWithOptionsGetResult = alice.call("blog", "get_post_with_options_latest", entryWithOptionsGet);
+
+     t.deepEqual(JSON.parse(entryWithOptionsGetResult.Ok.App[1]),updatePostV4Content);
+
+
+
+
+
```
Comment (lucksus) [inline app_spec/test/test.js]: Newlines

File: app_spec/test/test.js (hunk 1 above)
Comment (AshantiMutinta) [reply app_spec/test/test.js]: Yes, thank you for that!

**Pull request merged**